radicle-keystore/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "radicle-keystore"
+description = "Abstraction over storage of cryptographic keys within the Radicle stack"
+version = "0.1.0"
+authors = ["Monadic GmbH <radicle@monadic.xyz>"]
```

Comment:
Should we try to consistently migrate to dev@radicle.xyz?

Reply:
Did not know this exists. Such things should probably also be carved into the stone tablets of `radicle-decisions`

Reply:
Agreed.